app/src/main/java/org/schabi/newpipe/player/helper/PlayerHolder.java

```diff
@@ -31,6 +35,24 @@ private PlayerHolder() {
     private static MainPlayer playerService;
     private static VideoPlayerImpl player;
 
+    /**
+     * Returns the current {@link MainPlayer.PlayerType} of the {@link MainPlayer} service,
+     * otherwise `null` if no service running.
+     *
+     * @return Current PlayerType
+     */
+    @Nullable
+    public static MainPlayer.PlayerType getType() {
+        if (player == null) {
+            return null;
+        }
+
+        return player.videoPlayerSelected() ? MainPlayer.PlayerType.VIDEO
+                : player.popupPlayerSelected() ? MainPlayer.PlayerType.POPUP
+                : player.audioPlayerSelected() ? MainPlayer.PlayerType.AUDIO
+                : null;
```

Comment:
Add a method to `VideoPlayerImpl` that just returns the `playerType` field

Reply:
Alright, I keep the static method in `PlayerHolder`, so it will call `return player.getPlayerType()`.